kubernetes/ansible/roles/sunbird-monitoring/templates/sunbird_alert_html.tmpl

```diff
@@ -0,0 +1,124 @@
+{{ define "email.sunbird.html" }}
```

Comment:
I hope this is not an ansible template. In that case cant' we keep this in files/ directory ?